saylua/modules/forums/static-source/scss/main.scss

```diff
@@ -1,17 +1,13 @@
 @import 'assets/variables';
 @import 'assets/transitions';
 
-h1 {
-  font-weight: normal;
-  text-align: center;
-}
+$boardIconSize: 6rem;
 
 ul.forum-main-list {
   display: flex;
   flex-direction: column;
   width: 100%;
-  margin: 0;
-  margin-bottom: 5rem;
+  margin: 1em 0 5em;
```

Comment:
be explicit with that last one m8y, use all four if you're gonna do the first three.